docs/book/src/plugins/creating-plugins.md

```diff
@@ -64,3 +64,46 @@ Feel free to check the implementation under:
 
 - deploy-image: <https://github.com/kubernetes-sigs/kubebuilder/tree/v3.7.0/pkg/plugins/golang/deploy-image/v1alpha1>
 - grafana: <https://github.com/kubernetes-sigs/kubebuilder/tree/v3.7.0/pkg/plugins/optional/grafana/v1alpha>
+
+## Plugin scaffolding
+
+Your plugin may have the use case to add code on top of what is scaffolded by default with `create api`.
```

Comment:
It is not only about create api. The utils provided are valid for any scaffold/action and call made in a chain.